UnityProject/Assets/Scripts/Objects/Research/Artifacts/Effects/AreaEffects/AreaEffectBase.cs

```diff
@@ -65,27 +64,28 @@ protected virtual void TryEffectParts(PlayerScript toEffect)
 
 		protected virtual void TryEffectPlayer(PlayerScript toEffect)
 		{
-			float Armour = 0;
+			float totalAnomalyArmour = 0;
 			int partCount = 0;
 
 			List<BodyPart> parts = toEffect.playerHealth.SurfaceBodyParts.Shuffle().ToList(); //Without this it will almost always remove the same body part, i.e head
 
 			foreach (BodyPart part in parts)
 			{
-				float A2 = 0;
-				foreach (var armour in part.ClothingArmors)
+				float bodyPartTotalArmour = 0;
+				foreach (var anomalyArmour in part.ClothingArmors)
 				{
-					A2 += armour.Anomaly;
+					bodyPartTotalArmour += anomalyArmour.Anomaly;
 				}
 
-				Armour += Mathf.Clamp(A2, 0, 100);
+				totalAnomalyArmour += Mathf.Clamp(bodyPartTotalArmour, 0, 100);
```

Comment:
you are adding the clamped amount instead of assigning here, is that what you want?

Reply:
yeah that's all intended